modular_skyrat/modules/opposing_force/code/opposing_force_equipment.dm

```diff
@@ -0,0 +1,572 @@
+
+/datum/opposing_force_equipment/ammo/mg34
+	item_type = /obj/item/ammo_box/magazine/mg34
+
+/datum/opposing_force_equipment/ammo/cm762
+	item_type = /obj/item/ammo_box/magazine/cm762
+	description = "7.62 bullets in a ten round magazine for Cantanheim 7.62 rifle."
+
+/datum/opposing_force_equipment/ammo/a357
+	item_type = /obj/item/ammo_box/a357
+	description = "A seven-round .357 magnum speedloader for a revolver."
+
+/datum/opposing_force_equipment/ammo/a357match
+	item_type = /obj/item/ammo_box/a357/match
+	description = "A seven-round .357 magnum speedloader for a revolver, loaded with match-grade ammunition that bounces off walls several times."
+
+/datum/opposing_force_equipment/ammo/cfa_snub
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_snub
+
+/datum/opposing_force_equipment/ammo/cfa_snubap
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_snub/ap
+
+/datum/opposing_force_equipment/ammo/cfa_snubrubber
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_snub/rubber
+
+/datum/opposing_force_equipment/ammo/cfa_snubincendiary
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_snub/incendiary
+
+/datum/opposing_force_equipment/ammo/cfa_ruby
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_ruby
+
+/datum/opposing_force_equipment/ammo/cfa_rubyap
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_ruby/ap
+
+/datum/opposing_force_equipment/ammo/cfa_rubyrubber
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_ruby/rubber
+
+/datum/opposing_force_equipment/ammo/cfa_rubyhp
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_ruby/hp
+
+/datum/opposing_force_equipment/ammo/cfa_rubyincendiary
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_ruby/incendiary
+
+/datum/opposing_force_equipment/ammo/wildcat
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_wildcat
+
+/datum/opposing_force_equipment/ammo/wildcatap
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_wildcat/ap
+
+/datum/opposing_force_equipment/ammo/wildcatrubber
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_wildcat/rubber
+
+/datum/opposing_force_equipment/ammo/wildcatincendiary
+	item_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_wildcat/incendiary
+/**
+ * MELEE
+ */
+/datum/opposing_force_equipment/melee
+	category = OPFOR_EQUIPMENT_CATEGORY_MELEE_WEAPONS
+
+/datum/opposing_force_equipment/melee/switchblade
+	item_type = /obj/item/switchblade
+
+/datum/opposing_force_equipment/melee/metalbat
+	item_type = /obj/item/melee/baseball_bat/ablative
+	description = "A highly reflective baseball bat for when you need to crack skulls and run away from security in the same night."
+
+/datum/opposing_force_equipment/melee/esword
+	item_type = /obj/item/melee/energy/sword
+	description = "The energy sword is an edged weapon with a blade of pure energy. The sword is small enough to be \
+			pocketed when inactive. Activating it produces a loud, distinctive noise."
```

Comment:
The line break here is at the perfect distance pretty much. Also, when doing line breaks, just keep one indentation on the second line, not two.
```suggestion
	description = "The energy sword is an edged weapon with a blade of pure energy. The sword is small enough to be \
		pocketed when inactive. Activating it produces a loud, distinctive noise."
```